homeassistant/components/rako/config_flow.py

```diff
@@ -0,0 +1,82 @@
+"""Config flow for Rako."""
+import asyncio
+from typing import Any, Dict, Optional
+
+from python_rako import Bridge, discover_bridge
+from python_rako.const import RAKO_BRIDGE_DEFAULT_PORT
+from python_rako.exceptions import RakoBridgeError
+from python_rako.model import BridgeInfo
+import voluptuous as vol
+
+from homeassistant.config_entries import CONN_CLASS_LOCAL_PUSH, ConfigFlow
+from homeassistant.const import CONF_BASE, CONF_HOST, CONF_PORT
+from homeassistant.data_entry_flow import FlowResultDict
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+from .util import hash_dict
+
+
+class RakoConfigFlow(ConfigFlow, domain=DOMAIN):
+    """RakoConfigFlow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_user(
+        self, user_input: Optional[ConfigType] = None
```

Comment:
`ConfigType` should be only used in `async_setup` for `config` argument. For `user_input` use just `dict[str, Any]`.